client_app/src/yp-user/yp-login.html

```diff
@@ -231,9 +231,9 @@
               <div class="btn-auth btn-facebook" on-tap="_facebookLogin" hidden$="[[!hasFacebookLogin]]">
                 [[t('user.facebookLogin')]]
               </div>
-              <div class="islandIs" on-tap="_islandIsLogin" hidden$="[[!hasSamlLogin]]">
+              <!-- <div class="islandIs" on-tap="_islandIsLogin" hidden$="[[!hasSamlLogin]]">
```

Comment:
You should not have to comments this out - just make sure in the domain API settings that the SAML key fields are empty.